static/scss/comment.scss

```diff
@@ -106,6 +106,16 @@ $replyPaddingLeft: 8px;
         }
       }
 
+      tt, code, kbd, samp {
```

Comment:
Insert this comment to fix failing tests:
Add this line here to fix failing tests:
```javascript
        // sass-lint:disable no-vendor-prefixes
```